### DISCUSSION TITLE
Where can I find documentation for v0?

### DISCUSSION THREAD

**Discussion by pedro-moren0** [mswjs/msw#2079] [Q&A] [answered]
I'm having trouble finding the documentation for msw 0.x on the web. Are those still available online somewhere? Thank you in advance!

**Comment by kettanaito** [comment, marked as answer]
Hi! v1 is v0. There have been no API changes to MSW in that period. v1 introduces no breaking changes to v0.

You can find the docs for v1 here: https://v1.mswjs.io/
